src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -262,7 +262,11 @@ function positiveTestsStatusText(projection: Projection) {
 function hospitalOccupancyStatusText(projection: Projection) {
   const icuUtilization = projection.currentIcuUtilization;
   const currentlyInICU = projection.currentICUPatients;
-  if (icuUtilization === null || currentlyInICU == null) {
+  if (
+    icuUtilization === null ||
+    currentlyInICU == null ||
+    projection.typicallyFreeICUCapacity == null
```

Comment:
Note that I added this null check to make TypeScript happy, but if this is null, the other two values are likely null too, so we would have returned the "no data available" text anyway.